contrib/automation_tests/test_cases/live_tab.py

```diff
@@ -87,3 +87,20 @@ def _execute(self, function_name, min_calls=0, max_calls=pow(2, 33)-1):
         self.expect_true(min_calls <= call_count <= max_calls,
                          'Call count for function "%s" expected to be between %s and %s, was %s'
                          % (function_name, min_calls, max_calls, call_count))
+                         
+class VerifyOneFunctionWasCalled(LiveTabTestCase):
+    """
+    Verify that at least one of the functions matching the function name has 
+    received the given number of hits.
+    """
+    def _execute(self, function_name, min_calls=0, max_calls=pow(2, 33)-1):
```

Comment:
I tried to distinguish the parameter names between exact and partial matches in other tests - e.g. `function_name` vs `function_name_contains`. This would make the difference to `VerifyFunctionCallCount` clearer.

Reply:
Done - renamed parameter to `function_name_contains`.